src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -34,6 +34,12 @@ public class HubConnectionContext
         private long _lastSendTimestamp = Stopwatch.GetTimestamp();
         private ReadOnlyMemory<byte> _cachedPingMessage;
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="HubConnectionContext"/> class.
+        /// </summary>
+        /// <param name="connectionContext"></param>
```

Comment:
Should we just throw some bogus stuff in here? It seems weird to leave it completely blank